web/ui/react-app/src/pages/graph/GraphHelpers.test.ts

```diff
@@ -240,4 +240,46 @@ describe('GraphHelpers', () => {
       });
     });
   });
+
+  describe('emphasizeLine', () => {
```

Comment:
I tried to create a unit test for the new `emphasizeLine` function I added but I'm not exactly sure what would be the best way to do it.  Currently the test I wrote just makes sure no errors occur when it's run in expected use cases.  It doesn't actually make sure that the values are updated properly.  I wasn't sure what was the best way to go about this because in a chart, `getData` is a function that returns an object which `emphasizeLine` updates and that updates the chart.  Before I mock that sort of setup I wanted to see if someone thought it would be better to use an actual `plot` object instead (and if so, perhaps a suggestion on how to do that..).